DEVELOPER.md

```diff
@@ -333,7 +337,7 @@ Unit tests for the VerticaDB operator use the Go testing infrastructure. Some te
 ## e2e Tests
 
 > **IMPORTANT**
-> The e2e tests only run on operators that were [deployed as an object](#deployment-object).
+> The e2e tests only run on operators that were deployed as an object.
```

Comment:
This line should be removed. Deploy as an object is the only deployment option mentioned in this document (see this commit: https://github.com/vertica/vertica-kubernetes/commit/04390c61a1c811749a02b67b38d11d4c99bcf764).

Reply:
Will do